src/lib/components/particles/square-cut-corner-icon/index.jsx

```diff
@@ -0,0 +1,31 @@
+import defaultStyles from "./style.module.scss"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+
+export const SquareCutCornerIcon = ({cornerColor, squareColor, cornerPartyAbbreviation, squarePartyAbbreviation, styles}) => {
```

Comment:
is this the best way to offer an class / direct colour alternative?  - to have redundant arguments?